### DISCUSSION TITLE
Contributions on lxqt

### DISCUSSION THREAD

**Discussion by adjamhub** [lxqt/lxqt#1905] [General]
Hi all,
I'm planning to contribute to lxqt on a regular basis (more or less) and I'm writing here to share my ideas and hear what you think about.
I just did some PR (in space order: lxqt-powermanagement: lxqt/lxqt-powermanagement#225 and lxqt/lxqt-powermanagement#227 and lxqt/lximage-qt#310 and lxqt/lximage-qt#381) and I'd like to continue on the series of the "moving connect to Qt5 syntax" to take a look at the code overall. Moreover I'd like to continue working on lxqt-powermanagement, taking care of 2 things:

1. Implement separate idle watchers for AC and battery (lxqt/lxqt-powermanagement#217)
2. move lockscreenmanager (and its option in the GUI) from lxqt-session to lxqt-powermanagement (#1806)

That's it!
What do you think about?

**Comment by tsujan** [comment]
Your contributions are very welcome and really appreciated. LXQt needs active contributors like you.

**Reply by adjamhub** [reply]
Hi Tsujan, many thanks. Let's meet on my next PR then :)

**Reply by tsujan** [reply]
Looking forward to your next PRs and ideas.

**Comment by palinek** [comment]
Gteat to see new blood to LXQt. Welcome.

**Comment by stefonarch** [comment]
Fresh wind behind the boat, welcome!
